changelog_entry.yaml

```
@@ -0,0 +1,12 @@
+- bump: patch
+   changes:
+     added:
+     - SSDI
```

Then delete other bumps in the file (one bump per PR)
```suggestion
     - Calibration data: SSDI (2022 to 2032), Social Security (2017 to 2033), and federal income taxes by state (2021).
```